shopelectro/tests/helpers.py

```diff
@@ -82,3 +86,36 @@ def screen_fail(self, filename=''):
                 with open(f'screen__{filename}.png', 'wb') as f:
                     f.write(base64.b64decode(screen_b64.encode('ascii')))
             raise e
+
+    # @todo #300:60m Use or create clear selenium.WebElement behaviour system.from
+    #  Currently we return element from click method.
+    #  It's not clear design decision. Use or create your own behaviour system.
+    #  WebElement pipeline or smth.
```

Comment:
Maybe  [page objects pattern ](http://selenium-python.readthedocs.io/page-objects.html) will be helpful

Reply:
@artemiy312 , here we have problem not in test organizing in whole, but just in actions chaining.
So, i don't see this pattern helpful for this situation. But i'll add it to task desc, maybe implementor guy will find it helpful

TODO